packages/desktop-client/src/components/Modals.tsx

```diff
@@ -137,7 +137,7 @@ export default function Modals() {
           return (
             <ManageRulesModal
               modalProps={modalProps}
-              payeeId={options.payeeId}
+              payeeId={options?.payeeId}
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏is this change necessary? Doesn't the change in `ManageRules.tsx` already solve the problem?

Reply:
It's not strictly necessary, but `modals.d.ts` in `loot-core` lists `payeeId` as optional, so I thought it would make sense to match here!

Reply:
That makes sense. Thanks for explaining!